src/main/java/org/broadinstitute/hellbender/tools/coveragemodel/CoverageModelGlobalConstants.java

```diff
@@ -0,0 +1,149 @@
+package org.broadinstitute.hellbender.tools.coveragemodel;
+
+import java.io.Serializable;
+
+/**
+ * Global constants for coverage model package classes
+ *
+ * @author Mehrtash Babadi &lt;mehrtash@broadinstitute.org&gt;
+ */
+public final class CoverageModelGlobalConstants implements Serializable {
```

Comment:
That being said, I agree with perhaps more a thoughtful output structure. At the moment, the final model parameters are written to OUTPUT_PATH/model_final/, posteriors go to OUTPUT_PATH/posteriors_final/, and Viterbi segments go to OUTPUT_PATH/posteriors_final/segments/.